docs/start/smesher/post_1n.md

```diff
@@ -73,41 +73,35 @@ Run it with the following command:
 
 Where: `source_path` is the path to the node data you want to merge from and `target_path` is the path to the node data you want to merge to.
 
-:::note
-
-It is possible to merge nodes by hand too
-
-- Locate the `identity.key` files within the PoST data directories of each node.
-- Copy these files to the `./node_data/identities` directory on the node you're consolidating to.
-- Rename the key files respectively for easy identification of each identity.
-- Run: `sqlite3 target_node.sql` where `target_node.sql` is the database file of the node you're consolidating to.
-
-```attach '<source_path.sql>' as srcDB;
-BEGIN;
-insert into initial_post select * from srcDB.initial_post;
-insert into challenge select * from srcDB.challenge;
-insert into poet_registration select * from srcDB.poet_registration;
-insert into nipost select * from srcDB.nipost;
-COMMIT;
-detach srcDB;
-```
-
-It is however recommended only for advanced users as it does not cover all the corner cases and may lead to data corruption.
-
-:::
+> [!NOTE]
+> It is possible to merge nodes by hand too
+>
+> - Locate the `identity.key` files within the PoST data directories of each node.
+> - Copy these files to the `./node_data/identities` directory on the node you're consolidating to.
+> - Rename the key files respectively for easy identification of each identity.
+> - Run: `sqlite3 target_node.sql` where `target_node.sql` is the database file of the node you're consolidating to.
+>
+> ```attach '<source_path.sql>' as srcDB;
+> BEGIN;
+> insert into initial_post select * from srcDB.initial_post;
+> insert into challenge select * from srcDB.challenge;
+> insert into poet_registration select * from srcDB.poet_registration;
+> insert into nipost select * from srcDB.nipost;
+> COMMIT;
+> detach srcDB;
+> ```
+>
+> However, it is recommended only for advanced users as it does not cover all the corner cases and may lead to data corruption.
 
 3. **Configure PoST Services** :
 
-- For each identity, set up a PoST service that utilizes the existing PoST data linked to that identity. This ensures the node can continue to participate in the network without redoing the PoST.
+- For each identity, set up a PoST service that utilizes the existing PoST data linked to that identity. This ensures the node can continue participating in the network without redoing the PoST.
 - Detailed configuration steps can be found in the `post-service` [README](https://github.com/spacemeshos/post-rs/blob/main/service/README.md), guiding you through connecting each PoST service to your node.
 
 ## Operational Guide
 
-:::note
-
-This is just example, in real world you should use some kind of process manager like `systemd` or `supervisord` to manage your services. Or use some kind of orchestration tool like `kubernetes` or `docker-compose`.
-
-:::
+> [!WARNING]
```

Comment:
Same here :) it's GH syntax but we need a docusaurus instead;)

Reply:
yep, I just figured it out and wanted to revert